diff --git a/modulecmd.tcl b/modulecmd.tcl
--- a/modulecmd.tcl
+++ b/modulecmd.tcl
@@ -2205,7 +2205,7 @@ proc runModulerc {} {
          lappend rclist $env(MODULERCFILE)
       }
    }
-   if {[info exists env(MODULESHOME)]\
+   if {[info exists env(MODULESHOME)] && $env(MODULESHOME) ne ""\
       && [file isfile "$env(MODULESHOME)/etc/rc"]} {
       lappend rclist "$env(MODULESHOME)/etc/rc"
    }
-- 
2.9.4

