diff --git a/init/csh.in b/init/csh.in
--- a/init/csh.in
+++ b/init/csh.in
@@ -64,7 +64,7 @@ setenv MODULESHOME @prefix@
 @setmanpath@if ( $?MANPATH ) then
 @setmanpath@   set manpath = $MANPATH
 @setmanpath@else
-@setmanpath@   set manpath = `manpath -q`
+@setmanpath@   set manpath = `manpath`
 @setbinpath@endif
 @setmanpath@if ( ":${manpath}:" !~ '*:@mandir@:*' ) then
 @setmanpath@   if ( "$manpath" == '' ) then
-- 
2.9.4

