diff --git a/modulecmd.tcl b/modulecmd.tcl
--- a/modulecmd.tcl
+++ b/modulecmd.tcl
@@ -1859,16 +1859,7 @@ proc uname {what} {
             set result [info hostname]
          }
          release {
-            # on ubuntu get the CODENAME of the Distribution
-            if { [file isfile /etc/lsb-release]} {
-               set fd [open "/etc/lsb-release" "r"]
-               set a [read $fd]
-               regexp -nocase {DISTRIB_CODENAME=(\S+)(.*)}\
-                  $a matched res end
-               set result $res
-            } else {
-               set result $tcl_platform(osVersion)
-            }
+            set result $tcl_platform(osVersion)
          }
          domain {
             set result [exec /bin/domainname]
