diff --git a/testsuite/modules.10-use/080-access.exp b/testsuite/modules.10-use/080-access.exp
--- a/testsuite/modules.10-use/080-access.exp
+++ b/testsuite/modules.10-use/080-access.exp
@@ -21,6 +21,11 @@
 #
 ##############################################################################
 
+# cannot test access if superuser privileges are in use
+if {$tcl_platform(user) eq "root"} {
+    send_user "\tskipping access tests as tests are run by superuser\n"
+} else {
+
 #
 # Variables for this test
 #
@@ -105,3 +110,5 @@ unset unuse_modpath_sh
 unset unuse_modshare_sh
 
 unset module
+
+}
diff --git a/testsuite/modules.20-locate/080-access.exp b/testsuite/modules.20-locate/080-access.exp
--- a/testsuite/modules.20-locate/080-access.exp
+++ b/testsuite/modules.20-locate/080-access.exp
@@ -21,6 +21,11 @@
 #
 ##############################################################################
 
+# cannot test access if superuser privileges are in use
+if {$tcl_platform(user) eq "root"} {
+    send_user "\tskipping access tests as tests are run by superuser\n"
+} else {
+
 #
 # Variables for this test
 #
@@ -376,3 +381,5 @@ unset m15
 unset lmd15
 unset m16
 unset lmd16
+
+}
diff --git a/testsuite/modules.20-locate/081-access-rc.exp b/testsuite/modules.20-locate/081-access-rc.exp
--- a/testsuite/modules.20-locate/081-access-rc.exp
+++ b/testsuite/modules.20-locate/081-access-rc.exp
@@ -21,6 +21,11 @@
 #
 ##############################################################################
 
+# cannot test access if superuser privileges are in use
+if {$tcl_platform(user) eq "root"} {
+    send_user "\tskipping access tests as tests are run by superuser\n"
+} else {
+
 set modhome $env(TESTSUITEDIR)
 set etcdir "$modhome/etc"
 set etcmodrc "$etcdir/modulerc"
@@ -150,3 +155,5 @@ unset locate_msg
 unset locate2_msg
 unset err_sh
 unset hide_perms
+
+}
diff --git a/testsuite/modules.60-initx/080-access.exp b/testsuite/modules.60-initx/080-access.exp
--- a/testsuite/modules.60-initx/080-access.exp
+++ b/testsuite/modules.60-initx/080-access.exp
@@ -22,6 +22,11 @@
 #
 ##############################################################################
 
+# cannot test access if superuser privileges are in use
+if {$tcl_platform(user) eq "root"} {
+    send_user "\tskipping access tests as tests are run by superuser\n"
+} else {
+
 # restore regular .modules file for test
 set initf "$env(HOME)/.modules"
 set initfsave "${initf}.save"
@@ -76,3 +81,5 @@ unset write_perms
 
 unset initf
 unset initfsave
+
+}
diff --git a/testsuite/modules.61-coll/080-access.exp b/testsuite/modules.61-coll/080-access.exp
--- a/testsuite/modules.61-coll/080-access.exp
+++ b/testsuite/modules.61-coll/080-access.exp
@@ -21,6 +21,10 @@
 #
 ##############################################################################
 
+# cannot test access if superuser privileges are in use
+if {$tcl_platform(user) eq "root"} {
+    send_user "\tskipping access tests as tests are run by superuser\n"
+} else {
 
 #
 # Variables for this test
@@ -173,3 +177,4 @@ unset c1errmsg
 unset header_coll
 unset coll_list
 
+}
diff --git a/testsuite/modules.80-deep/080-access.exp b/testsuite/modules.80-deep/080-access.exp
--- a/testsuite/modules.80-deep/080-access.exp
+++ b/testsuite/modules.80-deep/080-access.exp
@@ -21,6 +21,11 @@
 #
 ##############################################################################
 
+# cannot test access if superuser privileges are in use
+if {$tcl_platform(user) eq "root"} {
+    send_user "\tskipping access tests as tests are run by superuser\n"
+} else {
+
 #
 # Variables for this test
 #
@@ -397,3 +402,5 @@ unset m17
 unset lmd17
 unset m18
 unset lmd18
+
+}
-- 
2.9.4

