diff --git a/testsuite/modules.50-cmds/095-uname.exp b/testsuite/modules.50-cmds/095-uname.exp
--- a/testsuite/modules.50-cmds/095-uname.exp
+++ b/testsuite/modules.50-cmds/095-uname.exp
@@ -47,6 +47,7 @@ regsub -all \# $u_version {\\\#} u_version
 regsub -all { } $u_version {\\\ } u_version
 regsub -all {\(} $u_domainname {\\\\\(} u_domainname
 regsub -all {\)} $u_domainname {\\\\\)} u_domainname
+regsub -all {\+} $u_release {\\+} u_release
 
 test_cmd_re "csh" "load $module/sysname" "$lmfmod;\n$lmmod;\n$lmf/sysname;\n$lm/sysname;\n$ts$u_sysname;\$"
 test_cmd_re "csh" "load $module/nodename" "$lmfmod;\n$lmmod;\n$lmf/nodename;\n$lm/nodename;\n$ts$u_nodename;\$"
-- 
2.9.4

